src/features/donation/components/single-recipient-allocation.tsx

```
@@ -332,6 +334,28 @@ export const DonationSingleRecipientAllocation: React.FC<
             )}
           />
         )}
+
+        {/* Cross-chain minimum amount warning with quick-fix button */}
+        {isCrossChainToken && crossChainMinAmount !== undefined && crossChainMinAmount > 0 && amount !== undefined && parseFloat(amount.toString()) > 0 && parseFloat(amount.toString()) < crossChainMinAmount && (
```

ESLint

[error] Replace `·crossChainMinAmount·!==·undefined·&&·crossChainMinAmount·>·0·&&·amount·!==·undefined·&&·parseFloat(amount.toString())·>·0·&&` with `⏎··········crossChainMinAmount·!==·undefined·&&⏎··········crossChainMinAmount·>·0·&&⏎··········amount·!==·undefined·&&⏎··········parseFloat(amount.toString())·>·0·&&⏎·········`

(prettier/prettier)